packages/http-client-java/emitter/src/options.ts

```diff
@@ -0,0 +1,26 @@
+import { JSONSchemaType } from "@typespec/compiler";
+
+// typespec-java has another "options.ts" file, with same "export".
+// If add/remove "export" here, please also check typespec-java in autorest.java repository.
+
+export const LIB_NAME = "@typespec/http-client-java";
+
+export interface DevOptions {
+  "generate-code-model"?: boolean;
+  debug?: boolean;
+  loglevel?: "off" | "debug" | "info" | "warn" | "error";
+  "java-temp-dir"?: string; // working directory for java codegen, e.g. transformed code-model file
+}
+
+export interface EmitterOptions {
+  "dev-options"?: DevOptions;
+}
+
+export const EmitterOptionsSchema: JSONSchemaType<EmitterOptions> = {
+  type: "object",
+  additionalProperties: true,
+  properties: {
+    "dev-options": { type: "object", additionalProperties: true, nullable: true },
```

Comment:
add description here as you'd want to see it in the docs 

```
../tspd/cmd/tspd.js doc . --enable-experimental --output-dir ./test --skip-js
```

to try generating the docs to see what it looks like

Reply:
and this is the config that `http-client-js` uses  but they are in the workspace so they can add it as a dependency(tspd is not published yet) 

```
"regen-docs": "tspd doc .  --enable-experimental  --output-dir ../../website/src/content/docs/docs/emitters/clients/http-client-js/reference --skip-js",
```

Reply:
I've added the npm script.

Since this script also update the emitter readme.md, I will use another PR for sync doc (we may want to keep some doc in current [readme](https://github.com/microsoft/typespec/blob/main/packages/http-client-java/README.md), e.g. the requirement for JDK and Maven)